crates/wasmer-provider/src/pipes.rs

```diff
@@ -0,0 +1,102 @@
+use std::{
+    convert::TryFrom,
+    fmt::Debug,
+    io::{self, Read, Seek, Write},
+};
+
+use anyhow::Error;
+use serde::{Deserialize, Serialize};
+use tokio::fs::File;
+use wasmer_wasi::{VirtualFile, WasiFsError};
+
+/// For piping stdio. Stores all output / input in a byte-vector.
+pub struct FilePipe {
+    file: std::fs::File,
+}
+
+impl Debug for FilePipe {
```

Comment:
Shouldn't `#[derive(Debug)]` be sufficient?

Reply:
It should yes, just a bad copy/paste I think